src/V3Timing.cpp

```diff
@@ -198,7 +217,11 @@ class TimingSuspendableVisitor final : public VNVisitor {
         if (!nodep->user5p()) nodep->user5p(new NeedsProcDepVtx{&m_procGraph, nodep, classp});
         return nodep->user5u().to<NeedsProcDepVtx*>();
     }
-    // Set timing flag of a node
+    // Add timing flag to a node
+    void addFlag(AstNode* const nodep, uint8_t flag) { nodep->user2(nodep->user2() | flag); }
+    // Remove timing flag from a node
+    void removeFlag(AstNode* const nodep, uint8_t flag) { nodep->user2(nodep->user2() & ~flag); }
```

Comment:
Should probably be `addFlags` and `removeFlags`, as you can set multiple flags (otherwise the flag argument would be the enum).

Reply:
`addFlag` is now renamed, I removed `removeFlags` -- it's unused and I forgot to do that before.